src/main/java/uk/gov/companieshouse/web/companydocuments/util/EnumerationConstantsMapper.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.web.companydocuments.util;
+
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EnumerationConstantsMapper {
+
+    private Map<String, String> companyTypes;
+
+    private Map<String, String> companyStatuses;
+
+    @Autowired
+    public EnumerationConstantsMapper(YamlResourceMapper yamlResourceMapper) {
+
+        Map<String, Object> enumerationConstants = yamlResourceMapper.fetchYamlFromFile("api-enumerations/constants.yml");
```

Comment:
This file is missing from the commits.

Reply:
That's because api-enumerations is a git submodule

Reply:
I must be doing something wrong then, as a pull on this branch, with a make clean dev results in something that won't boot.

Reply:
I've adapted the `Makefile` to suit, try now

Reply:
Fixed by your last commit to the Makefile.